master/docs/manual/cfg-statustargets.rst

```diff
@@ -1561,6 +1561,37 @@ In case any of `repoOwner`, `repoName` or `sha` returns `None`, `False` or empty
 
 You can define custom start and end build messages using the `startDescription` and `endDescription` optional interpolation arguments.
 
+StashStatusPush
+~~~~~~~~~~~~~~~
+
+.. @cindex StashStatusPush
```

Comment:
I actually don't know squat about rst, so I just followed the example for the GitHubStatus entry above.